docs/source/getting_started/handling_time.ipynb

```diff
@@ -0,0 +1,640 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cutoff_times = pd.DataFrame()\n",
+    "cutoff_times['customer_id'] = [1, 2, 3, 1]\n",
+    "cutoff_times['time'] = pd.to_datetime(['2014-1-1 04:00',\n",
+    "                             '2014-1-1 05:00',\n",
+    "                             '2014-1-1 06:00',\n",
+    "                             '2014-1-1 08:00'])\n",
+    "cutoff_times['label'] = [True, True, False, True]\n",
+    "cutoff_times\n",
+    "fm, features = ft.dfs(entityset=es,\n",
+    "                      target_dataframe_name='customers',\n",
+    "                      cutoff_time=cutoff_times,\n",
+    "                      cutoff_time_in_index=True)\n",
+    "fm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af390059",
+   "metadata": {},
+   "source": [
+    "We can now see that every row of the feature matrix is calculated at the corresponding time in the cutoff time dataframe. Because we calculate each row at a different time, it is possible to have a repeat customer. In this case, we calculated the feature vector for customer 1 at both ``04:00`` and ``08:00``.\n",
+    "\n",
+    "Training Window\n",
+    "---------------\n",
+    "\n",
+    "By default, all data up to and including the cutoff time is used. We can restrict the amount of historical data that is selected for calculations using a \"training window.\"\n",
+    "\n",
+    "Here's an example of using a two hour training window:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "240c7f4d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "window_fm, window_features = ft.dfs(entityset=es,\n",
+    "                                    target_dataframe_name=\"customers\",\n",
+    "                                    cutoff_time=cutoff_times,\n",
+    "                                    cutoff_time_in_index=True,\n",
+    "                                    training_window=\"2 hour\")\n",
+    "\n",
+    "window_fm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4efca117",
+   "metadata": {},
+   "source": [
+    "We can see that that the counts for the same feature are lower after we shorten the training window:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f044c9f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm[[\"COUNT(transactions)\"]]\n",
+    "\n",
+    "window_fm[[\"COUNT(transactions)\"]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d430509",
+   "metadata": {},
+   "source": [
+    "## Setting a Last Time Index\n",
+    "\n",
+    "The training window in Featuretools limits the amount of past data that can be used while calculating a particular feature vector. A row in the dataframe is filtered out if the value of its time index is either before or after the training window. This works for entities where a row occurs at a single point in time. However, a row can sometimes exist for a duration.\n",
```

Comment:
"This works for entities where a row occurs" > "This works for dataframes where a row occurs"